src/microbe_stage/Microbe.cs

```diff
@@ -2545,6 +2545,12 @@ private void OnBodyExitedEngulfArea(Node body)
         }
     }
 
+    private bool IsBindingForbidden(Microbe other)
```

Comment:
Usually in programming negative condition checks are harder to reason about, so maybe this should be `CanBind` and is used to check if all the binding checks *pass*

Reply:
The issue I had with naming was that there were already a few functions related in name, like `CheckBinding`, so I though `CanBind` may not be super speaking as a difference.

But I can definitely swap to positive condition instead, and rename it `IsAllowedToBind` (or `canBind` if you deem that the name will not be confused with `checkBinding`)